CMakeLists.txt

```diff
@@ -15,6 +15,9 @@ option (DO_INCLUDE_PLUGINS "Build subproject plugins" OFF)
 option (DO_INCLUDE_SDK "Build subproject sdk-cpp" OFF)
 
 option (DO_BUILD_TESTS "Set DO_BUILD_TESTS to OFF to skip building tests." ON)
+option (DO_ENABLE_EXCEPTIONS "Set DO_ENABLE_EXCEPTIONS to ON to disable compiling with exceptions" OFF)
```

Comment:
>OFF [](http://example.com/codeflow?start=99&length=3)

I would default this to ON and let Edge disable it.

Reply:
They probably don't need to do anything since they don't use cmake and by default DO_ENABLE_EXCEPTIONS will not be defined in their build system.